docs/concepts/cluster-administration/admission-control-webhooks.md

```diff
@@ -0,0 +1,79 @@
+# System Administrator’s Guide To Dynamic Admission Control
+
+As a system administrator, you'll likely want to enable various admission
+plugins to enact business policies. This document covers how to do that safely.
+
+## Registering
+
+Webhooks are registered with the ValidatingAdmissionWebhook and
+MutatingAdmissionWebhook API types. At the moment, it's essential that you
+register a webhook for all versions of an API--otherwise changes to a version
+which the webhook isn't registered for won't go through the webhook. (I.e., if
+you want a webhook on Deployments, you want to register for both the
+beta version in extensions and the v1 version in apps.)
+
+## Identity and permissions
+
+### What does your webhook trust?
+
+If the webhook is going to run in your cluster, the documentation for the
+webhook should tell you how to set up the identities it will trust. If the
+webhook is an external service, the service provider will have to give you some
+mechanism for demonstrating an identity (e.g., a token or a client-cert for
+mutual TLS).
+
+The configuration file for admission controllers allows you to specify a
+file in "kubeconfig" format. Each entry has a name; if the name "matches" (see
+below) a webhook, the contents of that entry will be used as the identity when
+calling out to that webhook. What does "match" mean? It is DNS based; e.g.
+`*.bar.example.com` matches `foo.bar.example.com`, but if a
+`foo.bar.example.com` section is present, it would take precendence. If the
+webhook is running in the cluster, the name and namespace of the service will be
+put into the form "name.namespace.svc" before matching.
+
+If your cluster is self-hosted, it likely makes sense to put these files into a
+secret.
+
+Important note: if you are running any extension apiservers, they also need to
+be configured with these files!
+
+### RBAC Rules
+
+Extension apiservers need to be able to get namespaces and read their labels in
+order to tell if they need to call a webhook. You'll need to make sure they have
+appropriate permissions.
+
+## Bootstrapping
+
+It is important to understand what a webhook will do and where it is
+implemented. For example, suppose a webhook is running via a deployment and
+modifies all pod objects. Then it is very important to be able to create the
+webhook's own pods when the webhook is not running, otherwise the cluster will
+be wedged. Do this by using the namespace label selector.
```

Comment:
How about adding a [link](https://github.com/kubernetes/api/blob/release-1.9/admissionregistration/v1beta1/types.go#L191) to the namespaceSelector field? A reader might not know what it it.